code/_onclick/hud/alert.dm

```diff
@@ -315,6 +315,23 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 	var/mob/living/carbon/C = owner
 	C.take(giver, receiving)
 
+/// Gives the player the option to succumb while in critical condition
+/obj/screen/alert/succumb
+	name = "Succumb"
+	desc = "Shuffle off this mortal coil."
+	icon_state = "succumb"
+
+/obj/screen/alert/succumb/Click()
+	var/mob/living/living_owner = owner
```

Comment:
Observers. usr can be different than the owner.

Reply:
Does Click() still fire for alerts when an observer clicks them?

Reply:
Yes, this is systemic issue where we use click as entry proc for screen objects but observing just pops someone else hud on your screen. There's probably a few bugs like that around still.